test/unit/region.spec.js

```diff
@@ -1168,6 +1168,20 @@ describe('region', function() {
     it('should return the region', function() {
       expect(this.region.destroy).to.have.returned(this.region);
     });
+
+    describe('when the region is already destroyed', function() {
+      it('should not reset the region', function() {
+        this.region.reset.reset();
```

Comment:
I am probably not looking hard enough but the extra reset in `this.region.reset.reset()` looks off. I can't find ref to the extra `reset`. I'll take a better look at this when I get the chance.